### PR TITLE
Release 4.13.0 OKAPI-1073

### PR DESCRIPTION
**Pull request by adamdickmeiss**

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -1,3 +1,40 @@
+## 4.13.0 2022-02-23
+
+Fixes:
+
+No major fixes in Okapi iself. Mostly just updates for dependency
+libraries.
+
+ * [OKAPI-1074](https://issues.folio.org/browse/OKAPI-1074) Vert.x 4.2.5
+ * [OKAPI-1069](https://issues.folio.org/browse/OKAPI-1069) Hazelcast security warning
```
Comment (julianladisch) [inline NEWS.md:9]: Better description:

> Remove unused Hazelcast dependency

File: NEWS.md [hunk 2]
```diff
@@ -1,3 +1,40 @@
...
+ * [OKAPI-902](https://issues.folio.org/browse/OKAPI-902) Update log4j2 configuration in Debian package
+ * [OKAPI-662](https://issues.folio.org/browse/OKAPI-662) Enhance install endpoint to report all
+version incompatible issues (not just the first)
+
+Changes:
+
+This version of Okapi happens to make a different order of modules during
+install as part of [OKAPI-662](https://issues.folio.org/browse/OKAPI-662) work.
+This is not an error, as multiple orders are ok as far as interface dependencies
+are concerned. Module mod-data-export-spring may break because of this.
```
Comment (julianladisch) [inline NEWS.md:32]:
```suggestion
are concerned. Module mod-data-export-spring <= 2.1.2 may break because of this.
```

**Review by julianladisch — APPROVED** [review]
Looks good, thanks!

File: NEWS.md (hunk 2 above)
Comment (adamdickmeiss) [reply NEWS.md]: ok

**Pull request merged**